handler.go

```diff
@@ -214,6 +217,35 @@ func extractDripRequest(params []any) (*drip, error) {
 }
 
 // healthcheckHandler is the default health check handler for the faucet
-func (f *Faucet) healthcheckHandler(w http.ResponseWriter, _ *http.Request) {
-	w.WriteHeader(http.StatusOK)
+func (f *Faucet) healthcheckHandler(_ http.ResponseWriter, r *http.Request) {
+	render.Status(r, http.StatusOK)
+}
+
+// readycheckHandler is the default ready check handler for the faucet
+func (f *Faucet) readycheckHandler(w http.ResponseWriter, r *http.Request) {
+	err := f.client.Ping()
+	if err != nil {
+		render.JSON(w, r, &response{
+			Message: fmt.Sprintf("node not ready: %s", err.Error()),
+			Info: map[string]any{
+				"time": time.Now().String(),
+			},
+		})
+
+		render.Status(r, http.StatusInternalServerError)
+
+		return
+	}
+
+	render.JSON(w, r, &response{
```

Comment:
The faucet should use the `NewJSONResponse` (JSON-RPC 2.0 responses), even for the health endpoint